api/provider/provider.go

```diff
@@ -6,10 +6,13 @@ import (
 	"encoding/json"
 	"io/ioutil"
 	"net/http"
+	"time"
 
 	"golang.org/x/oauth2"
 )
 
+const defaultTimeout = time.Second * 10
```

Comment:
I like having this, but it's only really a _default_ timeout if it can be overridden in some way

Reply:
hmm im not sure if this should be exposed via an env for it to be overridden since this is more of an internal config? would it be better if we use a `var` instead of a `const` instead?